index.html

```diff
@@ -13,29 +13,43 @@
 
 <body>
     <!-- Toolbar section -->
-    <header class="toolbar">
-        <div class="toolbar-menu">
-            <nav class="nav-bar">
-                <div class="hamburger">
-                    <i class="fa fa-bars"></i>
-                </div>
-                <ul class="menu-list">
-                    <li class="menu-item"><a href="#works-section">Portfolio</a></li>
-                    <li class="menu-item"><a href="#about-section">About</a></li>
-                    <li class="menu-item"><a href="#contact-section">Contact</a></li>
-                </ul>
-            </nav>
-
+ <header class="toolbar">
+  <div class="toolbar-menu">
+    <nav class="nav-bar">
+      <div class="hamburger" id="open-nav" onclick="openNav()">
+        <i class="fa fa-bars"></i>
+      </div>
+      <ul class="menu-list" id="menu-lists">
+        <li><a href="#works-section" class="menu-item">Portfolio</a></li>
+        <li><a href="#about-section" class="menu-item">About</a></li>
+        <li><a href="#contact-section" class="menu-item">Contact</a></li>
+      </ul>
+    </nav>
+  </div>
+</header>
+
+    
+    <!-- mobile-menu -->
+    <div class="mobile-menu hidden">
+        <div class="mobile-menu-container">
+            <div onclick="closeNav()" class="close-btn" id="close-nav">
```

Comment:
Semantics purposes and performance,try to avoid adding javascript events directly to your HTML , since the javascript file has already been added as `<script src="app.js"></script>` in your project.